### DISCUSSION TITLE
redirect configuration / p2 runtime data to non-default location

### DISCUSSION THREAD

**Discussion by gireeshpunathil** [eclipse-equinox/p2#522] [General]
use case: we want to install an eclipse based product in windows and share it among a number of users. hence we install it in a common folder (such as `C:\Program Files\...`). however, non-admin users are unable to run it properly, as the runtime configuration changes are not persisted because of the lack of permission in the installation location.

is there a way to redirect all the configuration / p2 related runtime persistence be redirected to another location, such as user's home folder?

**Comment by iloveeclipse** [comment]
Check Eclipse help.`-configuration` allows you to specify osgi config area that can be put under user specific location.

**Comment by laeubi** [comment]
This describes how you can share an installation with multiple users:

https://help.eclipse.org/latest/topic/org.eclipse.platform.doc.isv/reference/misc/multi_user_installs.html

**Comment by gireeshpunathil** [comment]
thanks @iloveeclipse and @laeubi ! this is really promising. will test and see if it addresses our scenario.
